### ISSUE TITLE
All props are listed as type 'unknown'

### ISSUE THREAD

**Issue by redders6600** [storybookjs/storybook#11226]
I'm using storybook 5.3.19 with a CRA project that I have migrated to typescript (it was previously JS and I was using `PropTypes`), however all of the props listed in my stories are displayed as 'other'. I am not sure where to start debugging this...

Example:

BasicSpan.tsx
```
type BasicSpanProps = {
  content: string;
};

export const BasicSpan: FC<BasicSpanProps> = ({ content = "test" }) => {
  return <span>{content}</span>;
};

export default WrappedDivider;
```

`BasicSpan.stories.tsx`
```
import { BasicSpan } from "./BasicSpan";

export const StoryOne = () => {
  return (
    <div>
      <BasicSpan content="something"></BasicSpan>
    </div>
  );
};

export default {
  title: "Basic span",
  component: BasicSpan,
};
```
![image](https://user-images.githubusercontent.com/2893500/85017499-791c5500-b163-11ea-9a8b-9d160c7941b1.png)

This is the output of `Docgen.parse`:
![image](https://user-images.githubusercontent.com/2893500/84995105-83c6f200-b143-11ea-83c5-6e0dfc15aa5b.png)

**Comment by shilman** [comment]
`Addon-info` is being superceded by `addon-docs`, which fixes a bunch of bugs and is easier to maintain. Please give it a try! https://medium.com/storybookjs/storybook-docspage-e185bc3622bf

**Comment by redders6600** [comment]
Hi @shilman thanks for the reply - I was actually using both but (the second screenshot is from docs I think), but I have just removed `addon-info`. I've now removed the confusing screenshot at the start of this issue so only the docs one is there, but I'm still having the same problem with docs - all props are showing as `unknown` regardless of how I define them.

As such it is probably worth re-labelling this issue to `addon: docs`.

**Comment by shilman** [comment]
Can you try upgrading to 6.0? We've smoothed out a lot of this there. Quick & dirty migration guide https://github.com/storybookjs/storybook/issues/9311

**Comment by redders6600** [comment]
@shilman thanks for your suggestion - it indirectly fixed my problem. From the migration guide, `npx npm-check-updates '/storybook/' -u` didn't upgrade storybook or any of its plugins (probably because I'm missing a flag to include beta versions, or something), but it _DID_ upgrade `@storybook/preset-create-react-app` which was on an old `2.x` version - I'd totally missed it because when I upgraded storybook I wrongly assumed all of the associated addonts/presets/plugins etc were using the same version number, so I used find/replace to do the update. Obviously the preset has its own versioning, so I completely missed it... 🤦 

So TL;DR: user error - outdated CRA preset.

Thanks for your help.

**Comment by shilman** [comment]
@redders6600 thanks for letting me know about the typo. Fixed! And glad you got it sorted!!
